packages/amplify-e2e-core/src/utils/rds.ts

```diff
@@ -35,11 +45,13 @@ const DEFAULT_SECURITY_GROUP = 'default';
 const IPIFY_URL = 'https://api.ipify.org/';
 const AWSCHECKIP_URL = 'https://checkip.amazonaws.com/';
 
+let clusterInfo: ClusterInfo;
```

Comment:
I'm not wild about this being a global -- it means that subsequent invocations of this method could mutate the value even though other processes may be relying on it. Why do we need to preserve this state here?

Reply:
You can just drop the database or I noticed that we don't drop the databases in CI flow (for reasons you mentioned earlier), so probably just keep the same behavior for now and we can add that later.